CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ Notable changes to this project are documented in this file. The format is based
 
 ## [Unreleased]
 
+Breaking changes:
+
+New features:
+Bugfixes:
```

Comment:
Can we add a newline here?

Reply:
Yeah, I've fixed this in my script. Thanks.